test/integration/kubernetes_deploy_test.rb

```diff
@@ -619,7 +619,6 @@ def test_failure_logs_from_unmanaged_pod_appear_in_summary_section
 
     assert_logs_match_all([
       "Failed to deploy 1 priority resource",
-      "SuccessfulMountVolume", # from an event
```

Comment:
1.11 isn't generating any events on this failure.

Reply:
Hm, I guess events aren't really the point of this test, so deleting is ok.